<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>yuhao</author>
 <class>bowers_Dialog</class>
 <widget class="QDialog" name="bowers_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PP Bowers</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Wells</string>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>141</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Prediction</string>
      </property>
      <widget class="QLineEdit" name="u_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.86</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>U:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="a_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>98</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.86</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Vmax:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="vmax_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>4800</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>B:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>A:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Unloading</string>
       </property>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>20</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>start:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="end_lineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>3500</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="buffer_lineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>end:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="start_lineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>3000</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="buffer_checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>buffer:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="predict_Button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_param_Button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icon/save_as_icon</normaloff>:/icon/save_as_icon</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Save</string>
      </property>
      <widget class="QPushButton" name="save_Button">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icon/save_icon</normaloff>:/icon/save_icon</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>_bowers</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocity:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="velocity_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="well_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="obp_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>OBP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameters:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="param_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Plot</string>
     </property>
     <widget class="QListWidget" name="pres_listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>121</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>141</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Horizons</string>
      </property>
      <widget class="QPushButton" name="plot_horizon_Button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>65</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icon/layer_icon</normaloff>:/icon/layer_icon</iconset>
       </property>
      </widget>
      <widget class="QListWidget" name="horizon_listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>171</height>
        </rect>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="litho_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Lithostatic</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="hydro_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hydrostatic</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
